fuzzers/dynamic_analysis/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "fuzzbench"
+version = "0.12.0"
+authors = ["Andrea Fioraldi <andreafioraldi@gmail.com>", "Dominik Maier <domenukk@gmail.com>"]
+edition = "2021"
+
+[features]
+default = ["std"]
+std = []
+no_link_main = ["libafl_targets/libfuzzer_no_link_main"]
+
+[profile.release]
+lto = true
+codegen-units = 1
+opt-level = 3
+debug = true
+
+[profile.release-fuzzbench]
+inherits = "release"
+debug = false
+strip = true
+
+[build-dependencies]
+cc = { version = "1.0", features = ["parallel"] }
+which = "4.4"
+
+[dependencies]
+env_logger = "0.10"
+once_cell = "1.19"
```

Comment:
once_cell is in the standard library now, no need to use the trait anymore

Reply:
the one i want to use LazyCell is nightly only

https://doc.rust-lang.org/std/cell/struct.LazyCell.html